src/mapgen_valleys.cpp

```diff
@@ -418,9 +332,8 @@ void MapgenValleys::calculateNoise()
 	}
 
 	for (s32 index = 0; index < csize.X * csize.Z; index++) {
```

Comment:
idk, its not really your fault its there, but there should at least not be new occurences of it added.

So maybe do a ctrl+f on the diff of this PR and replace index everywhere it gets declared inside a line with + at the left.